pysondb/cli.py

```diff
@@ -135,18 +147,72 @@ def verify_file(file_data, refer_keys, filename):
         print("The number keys in DB entries does not match")
 
 
-def main():
-    fire.Fire(
-        {
-            "create": create_if_not_exist,
-            "display": display,
-            "delete": delete,
-            "convert": convert,
-            "converttocsv": convert_db_to_csv,
-            "merge": merge,
-        }
+def set_parser():
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest="command")
+
+    create_parser = subparsers.add_parser("create", help="Create a new database file")
+    create_parser.add_argument("file_name", help="Name of the database file")
+
+    display_parser = subparsers.add_parser("display", help="Display a database file")
+    display_parser.add_argument(
+        "file_name", help="Name of the database file to display"
     )
 
+    delete_parser = subparsers.add_parser("delete", help="Delete a database file")
+    delete_parser.add_argument("file_name", help="Name of the database file to delete")
+
+    convert_parser = subparsers.add_parser(
+        "convert", help="Convert a csv file to a JSON database file"
+    )
+    convert_parser.add_argument("csv_file")
+    convert_parser.add_argument("json_db")
+
+    convert_to_csv_parser = subparsers.add_parser(
+        "converttocsv", help="Converts a JSON database to a csv file"
+    )
+    convert_to_csv_parser.add_argument("file_name", help="path of the target json file")
+    convert_to_csv_parser.add_argument(
+        "-t",
+        "--target_csv",
+        help="path of the converted csv, default : converted.csv",
+        default="converted.csv",
+    )
+
+    merge_parser = subparsers.add_parser("merge")
+    merge_parser.add_argument("p_file", help="The primary file")
+    merge_parser.add_argument("m_file", help="The file to combine with p_file")
+    merge_parser.add_argument(
+        "-o", "--output-file", help="The name of the output file, default: p_file"
+    )
+
+    args = parser.parse_args()
+
+    if args.command == "create":
+        create_if_not_exist(args.file_name)
+
+    elif args.command == "display":
+        display(args.file_name)
+
+    elif args.command == "delete":
+        delete(args.file_name)
+
+    elif args.command == "convert":
+        convert(args.csv_file, args.json_db)
+
+    elif args.command == "converttocsv":
+        convert_db_to_csv(args.file_name, args.target_csv)
+
+    elif args.command == "merge":
+        merge(args.p_file, args.m_file, args.output_file)
+
+    else:  # show help menu if the cli was started without an argument
+        parser.print_help()
+
+
+def main():
+    set_parser()
+
 
 if __name__ == "__main__":
     main()
```

Comment:
`raise SystemExit(main())` So that the exit codes for that last command is stored in the terminal session